software/spmd/simple_memory_test/main.c

```diff
@@ -0,0 +1,37 @@
+#include "bsg_manycore.h"
```

Comment:
Is this file supposed to be part of this PR?

Reply:
Yes

Reply:
Does it test something in this PR?

Reply:
I wrote this test to test that the profiler is working. It's not necessary

Reply:
I don't have an issue with adding another test - but it's not clear why the test is here. What does it test? How does it test the profiler?

The code can be removed, or stay as part of the PR. But it needs an explanation.

The rest of the PR is good to go

Reply:
It's just an easy program to analyze in the waveform and debug the profiler.

Reply:
If this is not checked in, I will need to spend time writing this program again next time I work on the profiler, .

Reply:
I have no doubt that this program is useful and I'm not asking that it be removed. However, like in #404 it is helpful if the test documents its intent: what it does, and why it matters. A few sentences is sufficient.

I have no issues with any of the rest of this PR. I was particularly happy that there's now a comment in testbenches/common/v/vanilla_core_profiler.v!